src/PrestaShopBundle/Controller/Admin/Sell/Address/AddressController.php

```diff
@@ -519,11 +520,15 @@ private function getErrorMessages(Exception $e): array
                     'The object cannot be loaded (the identifier is missing or invalid)',
                     'Admin.Notifications.Error'
                 ),
-                AddressConstraintException::INVALID_FIELDS => $this->trans(
+                AddressConstraintException::INVALID_REQUIRED_FIELDS => $this->trans(
                     'An error occurred when attempting to update the required fields.',
                     'Admin.Notifications.Error'
                 ),
             ],
+            InvalidAddressFieldException::class => $this->trans(
+                'Address contains invalid field values',
+                'Admin.Notifications.Error'
```

Comment:
@jolelievre is this string gonna be a generic one or does it have to be precise? If it is the latter case I would suggest _'Address fiels contains invalid values'_

Reply:
It's specific for the address domain, but it's kind of generic because it doesn't specify which field You only know "some" fields are invalid
I guess your suggestion is applicable then? `'Address fields contains invalid values`

Reply:
```suggestion
                'Address contains invalid values.',
                'Admin.Notifications.Error'
```

Reply:
@Junebyun done I just corrected the sentence with `Address fields contains invalid values` (since there are potentially several fields)

Reply:
Ok I just fixed it again it should be good now ^^

Reply:
Yes if there can be several fields '**_Address fields contain invalid values._**' is better! Thanks @jolelievre :)

Reply:
Alright I reverted it back 😉